config/schema.error.yaml

```diff
@@ -0,0 +1,567 @@
+2:
+    title: 'Cannot write to the `./app/etc/env.php` file'
+    suggestion: 'Deployment script cannot make required changes to the `/app/etc/env.php` file. Check your filesystem permissions.'
```

Comment:
I would add recommendation to check .magento.app.yaml that this directory is mounted (for integration) or create support ticket.

Reply:
All info for critical errors was automatically generated from devdocs.  There is a command for it `dev:generate:schema-error`. I added it to synchronize devdocs and this file.
I think we need to update devdocs too.